packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx

```diff
@@ -171,13 +172,17 @@ const PullRequestListPage: FC<PullRequestPageProps> = ({
           imageName="no-data-folder"
           title="No pull requests yet"
           description={[
-            t('views:noData.noPullRequests', 'There are no pull requests in this project yet.'),
+            t('views:noData.noPullRequests', `There are no pull requests in this ${repoId ? 'repo' : 'project'} yet.`),
```

Comment:
won't this default to the value in translation store and only if that is not available take the repId ? repo : project string here?

Reply:
Yes I realized that. I cleaned this up and added them as two separate strings instead.